package.json

```diff
@@ -83,18 +69,21 @@
     "karma-webpack": "2.0.3",
     "mocha": "3.2.0",
     "mockery": "2.0.0",
-    "mockgoose": "5.3.3",
     "node-sass": "4.1.1",
-    "npm-run-all": "4.0.2",
-    "react-addons-test-utils": "15.5.1",
+    "react-addons-test-utils": "15.6.2",
     "react-test-renderer": "15.5.4",
-    "sass-loader": "4.1.1",
+    "sass-loader": "^7.1.0",
     "sinon": "2.3.1",
     "sinon-chai": "2.10.0",
     "style-loader": "0.13.1",
     "supertest": "3.0.0",
-    "webpack": "2.2.1",
-    "webpack-dev-server": "2.4.1",
-    "webpack-merge": "3.0.0"
+    "webpack": "^4.16.4",
+    "webpack-cli": "^3.1.0",
+    "webpack-dev-middleware": "^3.1.3",
```

Comment:
`webpack-dev-server` uses `webpack-dev-middleware` in an express server to do what it does. This should be unnecessary

Reply:
we can install it globally or through the package. I can move it out if needed

Reply:
Sounds good. I just wasn't sure whether you wanted to merge these changes in before integrating CRA

Reply:
Well, i can start the integration now and submit another pull req when EVERYTHING is done. I just like to keep the merges incremental. What's would you think is best?

Reply:
Incremental is definitely better. But if so, I would prefer you fix these nitpicks before I approve